nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -329,6 +361,24 @@ class TfOpIdxMap {
       update_variables(op_node.getWeights());
       update_buffers(op_node.getWeights());
     }
+
+    auto update_model_io_to =
+      [&variable_map](const TfOpNode::Variables &variables,
+                      std::vector<int> &v) {
+        for (auto &variable : variables) {
+          auto index = variable_map.getIndex(variable);
+          v.push_back(index);
+        }
+      };
+
+    for (auto &op_node : nodes) {
+      if (op_node.isInput()) {
```

Comment:
isInput is checking if this layer is input layer of network model and isOutput too. so.. It confused little bit. How about change the name such as isInputLayer or so.

Reply:
I think the reason of confusion comes from we are not explicitly identifying tensor but only operation
But tf identifies tensors and operation.
I'll change to `isInputNode()` :)

Reply:
Updated